src/test/java/uk/gov/hmcts/reform/wataskmanagementapi/services/TaskManagementServiceTest.java

```diff
@@ -2806,6 +2819,123 @@ void should_throw_resource_not_found_exception_when_task_id_not_present() {
                 .saveTask(any());
         }
 
+        @Test
+        void should_throw_InvalidRequestException_when_NotesRequest_is_null() {
+            NotesRequest notesRequest = null;
+
+            InvalidRequestException exception = assertThrows(InvalidRequestException.class, () ->
+                taskManagementService.updateNotes("taskId", notesRequest));
+
+            assertThat(exception).isInstanceOf(InvalidRequestException.class);
+            assertNotNull(exception.getStatus());
+            assertThat(exception.getStatus().getStatusCode()).isEqualTo(BAD_REQUEST.getStatusCode());
+            assertThat(exception.getTitle()).isEqualTo("Bad Request");
+            assertThat(exception.getType().toString())
+                .isEqualTo("https://github.com/hmcts/wa-task-management-api/problem/bad-request");
+
+            verifyNoInteractions(cftTaskDatabaseService);
+        }
+
+        @Test
+        void should_throw_CustomConstraintViolationException_when_NoteResource_is_null() {
+            final NoteResource newNoteResource = null;
+            List<NoteResource> newNotes = new ArrayList<>();
+            newNotes.add(newNoteResource);
+            NotesRequest notesRequest = new NotesRequest(newNotes);
+
+
+            CustomConstraintViolationException exception = assertThrows(CustomConstraintViolationException.class, () ->
+                taskManagementService.updateNotes("taskId", notesRequest));
+
+            assertThat(exception).isInstanceOf(CustomConstraintViolationException.class);
+            assertNotNull(exception.getStatus());
+            assertThat(exception.getStatus().getStatusCode()).isEqualTo(BAD_REQUEST.getStatusCode());
```

Comment:
i think it would be better to put this code into a different and use it across the different tests.
i think that will improve readability of the code.